pyseir/cli.py

```diff
@@ -215,33 +249,41 @@ def _patch_substatepipeline_nola_infection_rate(
 
 def _write_pipeline_output(
     pipelines: List[Union[SubStatePipeline, StatePipeline]],
-    output_dir,
+    output_dir: str,
     output_interval_days: int = 4,
 ):
 
-    infection_rate_metric_df = pd.concat(p.infer_df for p in pipelines)
-
-    infection_rate_metric_df.to_csv(
-        path_or_buf=pyseir.utils.get_summary_artifact_path(
-            pyseir.utils.SummaryArtifact.RT_METRIC_COMBINED
-        ),
-        index=False,
+    infection_rate_metric_df = pd.concat(p.infer_df for p in pipelines).reset_index(drop=True)
+    timeseries_dataset = TimeseriesDataset(infection_rate_metric_df)
+    latest = timeseries_dataset.latest_values_object()
+    multiregion_rt = MultiRegionTimeseriesDataset.from_timeseries_and_latest(
+        timeseries_dataset, latest
     )
+    output_path = pathlib.Path(output_dir) / pyseir.utils.SummaryArtifact.RT_METRIC_COMBINED.value
+    multiregion_rt.to_csv(output_path)
+    root.info(f"Saving Rt results to {output_path}")
+
+    icu_df = pd.concat(p.icu_data.data for p in pipelines if p.icu_data).reset_index(drop=True)
+    timeseries_dataset = TimeseriesDataset(icu_df)
+    latest = timeseries_dataset.latest_values_object().data.set_index(CommonFields.LOCATION_ID)
+    multiregion_icu = MultiRegionTimeseriesDataset(icu_df, latest)
```

Comment:
Use one of the MultiRegionTimeseriesDataset "constructor" staticmethods to simplify tracking down the places that create these.